doc/connectivity/networking/api/net_pkt_mgmt.rst

```diff
@@ -1,10 +1,9 @@
-.. _net_buf_mgmt:
+.. _net_pkt_mgmt:
 
-Network Buffer Management
+Network Packet Management
```

Comment:
didn't we agree to remove 'Network' from the name in documentation?

> Agreement to promote it to a general library but keep the existing name temporarily for compatibility and adapt documentation and references, ie. remove networking from the name.

Reply:
I'd prefer to leaving naming intact for now and have a discussion/consensus on the full naming issue (API and description) before doing a follow up PR.

Reply:
This is something that was discussed and agreed on. Removing the 'Network' from the name will be less confusing and forward looking given that it now will live outside of networking. Do not see why this should be second step that we still have to discuss and agree on, see https://github.com/zephyrproject-rtos/zephyr/issues/36374#issuecomment-2123082283 for the minutes.

I am not talking about renaming the API and signatures which can happen later, this is about just dropping 'Networking' from the title of this API.

Does anyone disagree with that?

Reply:
> I am not talking about renaming the API and signatures which can happen later, this is about just dropping 'Networking' from the title of this API.

So you would just call the API "Buffers"?

Reply:
Buffy....